### PR TITLE
[#135664147] Network Account Info - Social Zombie

### PR DESCRIPTION
**Pull request by jonajonlee**
## [Removed Social Zombie from Kuss as discussed in this PR](https://github.com/influentialpublishers/kuss/pull/35)

- removed lib/social-zombie, express-socialzombie, and associated unit tests

### CODE REVIEW

File: lib/error.js [hunk 1]
```diff
@@ -48,10 +50,7 @@ NotFound.throw = R.curry((store, table, id) => {
 
 
 module.exports = {
-  STATUS: {
-    ERROR: STATUS_CODE_ERROR
-  , NOT_FOUND: STATUS_CODE_NOT_FOUND
-  }
+  STATUS_CODE
 , CussError
```
Comment (jxbadam) [inline lib/error.js]: Shouldn't this be KussError?

**Comment by jxbadam** [comment]
Also CI doesn't like your change (dropped test coverage)

**Comment by jxbadam** [comment]
:shipit:

**Review by scull7 — APPROVED** [review]
:shipit:

**Pull request merged**